libp2p/src/tutorials/ping.rs

```diff
@@ -307,16 +293,14 @@
 //!
 //! #[async_std::main]
 //! async fn main() -> Result<(), Box<dyn Error>> {
-//!     env_logger::init();
-//!     let local_key = identity::Keypair::generate_ed25519();
-//!     let local_peer_id = PeerId::from(local_key.public());
-//!
-//!     let transport = libp2p::development_transport(local_key).await?;
-//!
-//!     let behaviour = ping::Behaviour::default();
-//!
-//!     let mut swarm = SwarmBuilder::with_async_std_executor(transport, behaviour, local_peer_id)
-//!         .idle_connection_timeout(Duration::from_secs(30)) // Allows us to observe pings for 30 seconds.
```

Comment:
This tutorial is missing the idle connection timeout.

Reply:
Good catch. 3717d1fe2.